### PR TITLE
Java: add `xrevrange` command

### PR DESCRIPTION
**Pull request by acarbonetto**

### CODE REVIEW

File: java/integTest/src/test/java/glide/SharedCommandTests.java [hunk 1]
```diff
@@ -2948,9 +2948,6 @@ public void bzmpop_timeout_check(BaseClient client) {
                         ? RedisClient.CreateClient(commonClientConfig().build()).get()
                         : RedisClusterClient.CreateClient(commonClusterClientConfig().build()).get()) {
 
-            // ensure that commands doesn't time out even if timeout > request timeout
-            assertNull(testClient.bzmpop(new String[] {key}, MAX, 1).get());
```
Comment (Yury-Fridlyand) [inline java/integTest/src/test/java/glide/SharedCommandTests.java:2952]: why?

File: java/integTest/src/test/java/glide/SharedCommandTests.java [hunk 2]
```diff
@@ -3104,11 +3101,23 @@ public void xrange(BaseClient client) {
         assertNotNull(result.get(streamId1));
         assertNotNull(result.get(streamId2));
 
+        // get everything from the stream using a reverse range search
```
Comment (Yury-Fridlyand) [inline java/integTest/src/test/java/glide/SharedCommandTests.java:3104]: Please rename the test case (line 3072)

**Review by Yury-Fridlyand — COMMENTED** [review]
I skipped detailed review, assuming you just copied all from `xrange` and updated the doc.

File: java/integTest/src/test/java/glide/SharedCommandTests.java (hunk 1 above)
Comment (acarbonetto) [reply java/integTest/src/test/java/glide/SharedCommandTests.java:2952]: I don't know... must be a merging conflict.  Will revert.

Comment (acarbonetto) [reply java/integTest/src/test/java/glide/SharedCommandTests.java:2952]: Oh - I was showing something to Chloe on this test.

**Review by Yury-Fridlyand — APPROVED** [review]

**Pull request merged**